agent/src/softwarecontaineragent.h

```diff
@@ -79,7 +80,7 @@ class SoftwareContainerAgentError : public SoftwareContainerError
 class SoftwareContainerAgent
 {
     LOG_DECLARE_CLASS_CONTEXT("SCA", "SoftwareContainerAgent");
-    typedef std::shared_ptr<SoftwareContainer> SoftwareContainerPtr;
+    typedef std::shared_ptr<ContainerAgentInterface> SoftwareContainerPtr;
```

Comment:
Actually in the first idea the name of that class was ContainerAbstractInterface. Yet there is another ContainerAbstractInterface which connect software container to real back-end impelmentation. So not to create confusion i wanted to make it clear that this is interface used by agent. But may be it can be SoftwareContainerAbstractInterface. What do you think